src/components/LoginNavbar.vue

```diff
@@ -16,6 +16,22 @@
 		computed: {
 			offlineModule() {
 				return this.$store.getters.getOfflineModules
+			},
+			orderedModules(){
+				return this.offlineModule.sort((a, b) => { return b.priorityNumber < a.priorityNumber;});
+			}
+		},
+		methods:{
+			moduleName(traductions){
+				var moduleName = traductions.en;
+				if(this.$i18n.locale == 'fr'){
+					moduleName = traductions.fr;
+				}else if( this.$i18n.locale == 'es'){
+					moduleName = traductions.es;
+				}else if( this.$i18n.locale == 'en'){
+					moduleName = traductions.en;
+				}
```

Comment:
moduleName = traductions[this.$i18n.locale] ? traductions[this.$i18n.locale] : moduleName